Microsoft.Toolkit.Uwp.UI.Controls/SlidableListItem/SlidableListItem.cs

```diff
@@ -256,6 +268,12 @@ private void ContentGrid_ManipulationStarted(object sender, ManipulationStartedR
                 _contentStoryboard.Children.Add(_contentAnimation);
             }
 
+            if (!_contentStoryboardCompletedIsSubscribed)
```

Comment:
Instead of keeping a variable if the event has been subscribed, subscribe to it when creating the storyboard and then in the loaded event check if the storyboard is not null and subscribe
```
        private void OnLoaded(object sender, RoutedEventArgs e)
        {
            if (_contentStoryboard != null)
            {
                _contentStoryboard.Completed += ContentStoryboard_Completed;
            }
        }

        private void OnUnloaded(object sender, RoutedEventArgs e)
        {
            if (_contentStoryboard != null)
            {
                _contentStoryboard.Completed -= ContentStoryboard_Completed;
            }
        }
```